src/include/indexlr-minimize.h

```diff
@@ -33,6 +35,22 @@ struct HashData
 	char strand;
 };
 
+struct find_HashData
```

Comment:
Seems redundant, should be merged with HashData.

Reply:
Removed the code as new implementation no longer requires it